scripts/ccpp_prebuild_config_FV3.py

```diff
@@ -86,6 +88,11 @@
     'ccpp/physics/physics/gwdps.f'                        : [ 'slow_physics' ],
     'ccpp/physics/physics/samfdeepcnv.f'                  : [ 'slow_physics' ],
     'ccpp/physics/physics/samfshalcnv.f'                  : [ 'slow_physics' ],
+#   'ccpp/physics/physics/cu_gf_deep.F90'	          : [ 'slow_physics' ],
```

Comment:
I notice that you are using tabs instead of spaces. Please replace all tabs with four spaces